core/src/banking_stage.rs

```diff
@@ -411,9 +410,9 @@ impl BankingStage {
             .unwrap_or(false);
         // Many banks that process transactions in parallel.
         let bank_thread_hdls: Vec<JoinHandle<()>> = (0..num_threads)
-            .map(|i| {
+            .map(|id| {
```

Comment:
I also changed this single character variable to `id` since that's how it's being used

Reply:
may as well call it `thread_id`